src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -354,8 +355,11 @@ public String captureGesture() {
 
   public String captureGesture(String gestureName) {
     StringBuffer script = new StringBuffer();
+    DateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
```

Comment:
It's better to create this object once as a private member variable of the class, rather than re-creating it each time it's being used.